cmd/edgex-secrets-config/Attribution.txt

```diff
@@ -0,0 +1,176 @@
+The following open source projects are referenced by Security Proxy Setup:
+
+pkg/errors (BSD-2) https://github.com/pkg/errors
+https://github.com/pkg/errors/blob/master/LICENSE
+
+gorilla/mux (BSD-3) - https://github.com/gorilla/mux
+https://github.com/gorilla/mux/blob/master/LICENSE
+
+globalsign/mgo (unspecified) - https://github.com/globalsign/mgo
+https://github.com/globalsign/mgo/blob/master/LICENSE
+
+pebbe/zmq4 (BSD-2) https://github.com/pebbe/zmq4
+https://github.com/pebbe/zmq4/blob/master/LICENSE.txt
+
+go-kit/kit (MIT) github.com/go-kit/kit
+https://github.com/go-kit/kit/blob/master/LICENSE
+
+go-logfmt/logfmt (MIT) https://github.com/go-logfmt/logfmt
+https://github.com/go-logfmt/logfmt/blob/master/LICENSE
+
```

Comment:
Honestly, I don't know how to generate it.  I know that bin/test-attribution-txt.sh will complain if things are missing, but I don't know what tool to run to sync it up.  I just copied it from proxy-setup.

Reply:
Yes, those are dependent packages.  bin/test-attribution-txt.sh will also complain, I think,  if you have that is no longer needed.

Reply:
OK, it looks like they're all pretty much the same (i.e. every service's Attribution.txt), which I don't think is quite right. Each service has a unique set of imports and the idea is that the container should include an Attribution.txt which documents the imports and associated licenses. It looks like the current approach we've taken generates an Attribution.txt file which includes all imports for every service, so it's essentially a superset. I'm not sure this is exactly what was intended, but it's better than nothing... 

@anonymouse64 as the original author of the script, any thoughts?

Reply:
@tonyespy , this is because the mono-repo has a single go.mod file. If we wanted these to have just the dependencies that each service uses then we would need to put a go.mod file in the `cmd` folder for each one, which may be better way?? But for now, it is what it is... ;-)

Reply:
@lenny-intel ah yea, the joys of a mono-repo, thanks for clarifying!